### ISSUE TITLE
session reset after clicking on public page link in scp

### ISSUE THREAD

**Issue by jeffchulg** [osTicket/osTicket-1.7#681]
Hi, as mentioned in #667 :

For sure : So I clic public/test-php
![95310824-0aeb-11e3-8d22-0aa9aa11ab1d](https://f.cloud.github.com/assets/1788348/1031526/eeac8300-0ee3-11e3-9d88-9661acec4ba5.png)

Then I come to this page :

![c3e85e4c-0aeb-11e3-824b-b859e169ee1f](https://f.cloud.github.com/assets/1788348/1031527/f1a08a7a-0ee3-11e3-9600-65a2eb9cfcd7.png)

And finally go back to the osticket scp ( myosticket/scp )
![23c3984e-0bab-11e3-937c-8599262a8f87](https://f.cloud.github.com/assets/1788348/1031529/f53df442-0ee3-11e3-854d-b948f377eb2a.jpg)

**Comment by greezybacon** [comment]
@jeffchulg, where'd you get that last screenshot? It isn't version 1.7. Are you sure you're not accidentally visiting another osTicket installation on the same server?

**Comment by jeffchulg** [comment]
the last one was not... lazyness... lol
But the others was from my 1.7.1 RC1 installation

**Comment by greezybacon** [comment]
Should I close the issue?

**Comment by jeffchulg** [comment]
no, it's a real issue I experienced after the changes you asked me to do in #667 
so, I think it's a real issue...

**Comment by greezybacon** [comment]
@jeffchulg, I have no idea what you're suggesting is broken. The screenshot you posted is not from osTicket 1.7.0 or 1.7.1-rc1. Is it that the system is logging you out? Does the back button + refresh do the same?

**Comment by jeffchulg** [comment]
@greezybacon , as I mentioned, I previously realized (in the issue #667 ) that I had mistaken the image I uploaded. I was out of the office when I uploaded the screenshot of the login page. Sorry for not taking another screenshot and uploading the good one. 

By the way, I have finally found out what was the problem I was facing : 
I'm using osticket in https mode, but I had not set the "Helpdesk URL" with https. I've just set it with https and the problem is resolved. 
So it's a setting issue, not an application issue. Sorry.
